app-editors/atom/atom-1.16.0.ebuild

```diff
@@ -1,16 +1,17 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
+# $Id$
```

Comment:
These headers are gone for good. Don't add them back please. See 22fd7b2d4764535bd3afc37728df1015b0294467.